matcher/service/config/celery_config.py

```diff
@@ -0,0 +1,24 @@
+from kombu import Exchange, Queue
+from kombu.serialization import registry
+
+task_default_queue = "default"
+task_queues = (
+    Queue("matching", routing_key="matcher.service.celery_tasks.run_matching"),
+    Queue(
+        "deployment", routing_key="matcher.service.celery_tasks.run_deployment"
+    ),
+    Queue(
+        "failure", routing_key="matcher.service.celery_tasks.set_error_status"
+    ),
+)
+task_ignore_result = False
+broker_url = "redis://localhost:6379/10"
```

Comment:
Does the 10 here indicate that the redis db 10 is being used?

Reply:
Right, I asked Purujit to use 10 so it doesn't overlap with the database numbers of the API.